Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM python:3.5
-WORKDIR /code
-ADD . /code
-RUN pip install cliquet[postgresql,monitoring]
-RUN pip install -e .
-RUN kinto --backend=memory init
-CMD kinto start
+# Mozilla Kinto server
+FROM stackbrew/debian:sid
```

Comment:
You just download to build the image but since docker-hub build it for you actually just download a small image.